src/Mechanisms/HandleComponents/Synthesizers/Tests/FloatFormSynthUnitTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Livewire\Mechanisms\HandleComponents\Synthesizers\Tests;
+
+use Livewire\Component;
+use Livewire\Form;
+use Livewire\Livewire;
+
+class FloatFormSynthUnitTest extends \Tests\TestCase
```

Comment:
Can you merge these tests into the FloatSynthUnitTest file and instead add like a `hydrated_in_a_form` to the test names or something like that?

Reply:
As I wrote in the PR description, I wasn't sure whether explicit tests of hydrating forms were desired, as they are not part of the other synthesizer Unit Tests. In my opinion, however, this should definitely be tested explicitly.

Done in commit [feb8cb8](https://github.com/livewire/livewire/pull/8126/commits/feb8cb8c0d7c71fcae4fa4f0c7f517e78d9ab33a)